GenerateDocs.ps1

```diff
@@ -498,7 +498,7 @@ Some environment variables are set based on options that are passed, configurati
 #### Business Edition Variables
 
  * ChocolateyInstallArgumentsSensitive - Encrypted arguments passed from command line `--install-arguments-sensitive` that are not logged anywhere. (0.10.1+ and licensed editions 1.6.0+)
- * ChocolateyPackageParametersSensitive - Package parameters passed from command line `--package-parameters-senstivite` that are not logged anywhere.  (0.10.1+ and licensed editions 1.6.0+)
+ * ChocolateyPackageParametersSensitive - Package parameters passed from command line `--package-parameters-sensitive` that are not logged anywhere.  (0.10.1+ and licensed editions 1.6.0+)

```

Comment:
Pretty sure this is just a typo that doesn't accurately reflect the state of the world.